utils/crypto/bls/public.go

```diff
@@ -41,6 +42,20 @@ func PublicKeyFromBytes(pkBytes []byte) (*PublicKey, error) {
 	return pk, nil
 }
 
+// PublicKeyToUncompressedBytes parses the uncompressed big-endian format
+// of the public key into a public key. It is assumed that the provided bytes
+// are valid.
```

Comment:
also would this be "returns the uncompressed big-endian format of the public key"?

Reply:
yes